common/txmgr/types/tx_store.go

```diff
@@ -33,12 +33,6 @@ type TxStore[
 	TxHistoryReaper[CHAIN_ID]
 	TransactionStore[ADDR, CHAIN_ID, TX_HASH, BLOCK_HASH, SEQ, FEE]
 
-	// Find confirmed txes beyond the minConfirmations param that require callback but have not yet been signaled
-	FindTxesPendingCallback(ctx context.Context, blockNum int64, chainID CHAIN_ID) (receiptsPlus []ReceiptPlus[R], err error)
-	// Update tx to mark that its callback has been signaled
-	UpdateTxCallbackCompleted(ctx context.Context, pipelineTaskRunRid uuid.UUID, chainId CHAIN_ID) error
-	SaveFetchedReceipts(ctx context.Context, r []R, state TxState, errorMsg *string, chainID CHAIN_ID) error
-
```

Comment:
Let's keep these for now so we can keep track of every tx_store method required by the current TXM version and quickly compare it with TXMv2

Reply:
Couldn't we still do that through the git history? Feel like I should clean this out with this change since they aren't being used anymore. Otherwise we'd also have to maintain the methods in `evm_tx_store.go` which seems excessive

Reply:
These methods are required by the Finalizer, so they have to be maintained by the `evm_tx_store.go`. I think the debate is whether Finalizer's tx store interface is enough, or whether we should also declare them in this interface for housekeeping.

Reply:
Oh sorry I thought you meant keeping all of the tx store methods I removed. Missed that you were specifically referring to these 3. I don't mind adding these back.